.github/workflows/release-drafter.yml

```diff
@@ -11,8 +11,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Drafts your next Release notes as Pull Requests are merged into "master"
-      - uses: release-drafter/release-drafter@v5
+      - name: Drafting release
+        id: release_drafter
+        uses: release-drafter/release-drafter@v5
         with:
           config-name: release-drafter.yml
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.RELEASEDRAFTER_PAT }}
+      - name: Write release to file
+        run: |
+          CONTENT=${{ steps.release_drafter.outputs.body }}
```

Comment:
Oh! There is an active bug here in the `meshery` repo. In the meantime, while that is being fixed, we can pull from https://github.com/layer5io/meshery-istio/tree/master/.github

Reply:
Removed that step, the rest of the workflow is same. Kept the name of release_drafter step (meshrey-istio workflow has no name) so that CI dashboard is readable. Sounds good?

Reply:
Yes, indeed. Good deal. ✔️